plugins/GigPlayer/GigPlayer.cpp

```diff
@@ -576,14 +588,14 @@ void GigInstrument::loadSample( GigSample& sample, sampleFrame* sampleData, f_cn
 	{
 		sample.sample->SetPos( sample.pos );
 
-		unsigned long size = sample.sample->Read( &buffer, samples ) * sample.sample->FrameSize;
-		std::memset( (int8_t*) &buffer + size, 0, allocationsize - size );
+		unsigned long size = sample.sample->Read( buffer, samples ) * sample.sample->FrameSize;
+		std::memset( (int8_t*) buffer + size, 0, allocationsize - size );
```

Comment:
```suggestion
		const auto size = sample.sample->Read(buffer, samples) * sample.sample->FrameSize;
		std::memset(buffer + size, 0, allocationsize - size);
```

Perhaps this can be written a bit better?

Reply:
There are a lot of changes that should happen here in GigPlayer, but I decided to leave this as a minimum viable PR to get it to build on Windows. Definitely something that should be updated in a followup.